js/mercado.js

```diff
@@ -751,8 +753,8 @@ module.exports = class mercado extends Exchange {
         await this.loadMarkets ();
         const market = this.market (symbol);
         const request = {
-            'precision': this.timeframes[timeframe],
-            'coin': market['id'].toLowerCase (),
+            'resolution': this.timeframes[timeframe],
```

Comment:
I think it's safer to do this instead: `this.safeString (this.timeframes, timeframe, timeframe)` just in case the exchange introduces a new frame

Reply:
u'r right, but the reason is that everywhere we already use `this.timeframes[timeframe]`, so that exact edit should be of other scope (not in this PR) and I will make a separate PR which would fix that issue across all exchanges. but again, we should skip that detail from this PR.